docs/CH32V30x.md

```diff
@@ -1,8 +1,28 @@
 # CH32V30x
 
-- CH32V303
-- CH32V305
-- CH32V307
+- CH32V303xBT6: RISC-V4F, 128KB Flash, 32KB SRAM, 144MHz, USB2.0 FS
```

Comment:
Wrong description.
All CH32V30x has 480K Flash, only "zero-wait" Flash section varies.

Reply:
Thanks for the correction, I have already noticed it in the main Readme. I'll push a fix later today (my timezone).